lms/djangoapps/discussion/tasks.py

```diff
@@ -113,8 +113,17 @@ def _is_not_subcomment(comment_id):
 
 def _is_first_comment(comment_id, thread_id):
     thread = cc.Thread.find(id=thread_id).retrieve(with_responses=True)
-    first_comment = thread.children[0]
-    return first_comment.get('id') == comment_id
+    if getattr(thread, 'children', None):
+        first_comment = thread.children[0]
+        return first_comment.get('id') == comment_id
+    else:
+        log.info(
+            "EDUCATOR-3385: No child exists for thread_id %s | course_id %s | username %s ",
+            thread['id'],
```

Comment:
Consider using `thread.get('id')` so this logging won't encounter data missing problems?

Reply:
sure.